<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Motors</class>
 <widget class="QMainWindow" name="Motors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="MotorTabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="motorwidget">
     <attribute name="title">
      <string>Motors</string>
     </attribute>
     <widget class="QFrame" name="motorsdisplay">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>620</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="MyTextBrowser" name="x_display">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="MyTextBrowser" name="y_display">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>60</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="MyTextBrowser" name="z_display">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>60</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="MyTextBrowser" name="pitch_display">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="MyTextBrowser" name="yaw_display">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>130</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="MyTextBrowser" name="roll_display">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>130</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="MyLabel" name="x_label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X motor</string>
       </property>
      </widget>
      <widget class="MyLabel" name="y_label">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Y motor</string>
       </property>
      </widget>
      <widget class="MyLabel" name="z_label">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>30</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Z motor</string>
       </property>
      </widget>
      <widget class="MyLabel" name="yaw_label">
       <property name="geometry">
        <rect>
         <x>479</x>
         <y>110</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Yaw motor</string>
       </property>
      </widget>
      <widget class="MyLabel" name="roll_label">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>110</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Roll motor</string>
       </property>
      </widget>
      <widget class="MyLabel" name="pitch_label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pitch motor</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="motoractions">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>620</width>
        <height>171</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="MyLabel" name="Move_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Move</string>
       </property>
      </widget>
      <widget class="MyTextEdit" name="distance">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>70</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="motor_selector">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>91</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
      <widget class="MyLabel" name="units">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>70</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>units</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="move_abs">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>50</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="accessibleName">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Move Absolute</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="move_rel">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>80</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="accessibleName">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Move Relative</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="utilitieswidget">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Utilities</string>
     </attribute>
     <widget class="QFrame" name="system_utilities">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>611</width>
        <height>171</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="MyPushButton" name="connect">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>130</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>0</y>
         <width>428</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="MyPushButton" name="ResetAll">
          <property name="text">
           <string>Reset All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyPushButton" name="HomeGantry">
          <property name="text">
           <string>Home Gantry</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>126</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>50</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="MyLabel" name="sys_label">
          <property name="text">
           <string>System Utilities</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyLabel" name="conn_label">
          <property name="text">
           <string>PMAC connection</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>471</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <property name="horizontalSpacing">
         <number>50</number>
        </property>
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="MyLabel" name="IP_label">
          <property name="text">
           <string>PMAC IP</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="MyLabel" name="uname_label">
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="MyLabel" name="pword_label">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="MyTextEdit" name="IP_field">
          <property name="toolTip">
           <string notr="true"/>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="MyTextEdit" name="uname_field"/>
        </item>
        <item row="1" column="2">
         <widget class="MyTextEdit" name="pword_field"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="motor_utilities">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>611</width>
        <height>171</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QComboBox" name="motor_selector_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>91</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="mutil_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Motor Utilities</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="set_display">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="MyPushButton" name="getspeed">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>70</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Get Speed</string>
       </property>
      </widget>
      <widget class="MyPushButton" name="setspeed">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>70</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Set Speed</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>units</string>
       </property>
      </widget>
      <widget class="QLabel" name="mutil_label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Jog Speed</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="MyLabel" name="pmac_display">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>390</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: red;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyLabel" name="sh_display">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: red;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyLabel" name="pmac_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pmac comms</string>
    </property>
   </widget>
   <widget class="MyLabel" name="sh_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>System homed</string>
    </property>
   </widget>
   <widget class="MyPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">Graphics.BaseClasses</header>
  </customwidget>
  <customwidget>
   <class>MyTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header location="global">Graphics.BaseClasses</header>
  </customwidget>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">Graphics.BaseClasses</header>
  </customwidget>
  <customwidget>
   <class>MyTextEdit</class>
   <extends>QTextEdit</extends>
   <header>Graphics.BaseClasses</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
